core/primitives/src/version.rs

```diff
@@ -125,6 +125,8 @@ pub enum ProtocolFeature {
     AltBn128,
     #[cfg(feature = "protocol_feature_chunk_only_producers")]
     ChunkOnlyProducers,
+    #[cfg(feature = "protocol_feature_new_validator_selection_algorithm")]
+    NewValidatorSelectionAlgorithm,
```

Comment:
please add comment describing the feature.

Also - we should come up with a better name - as 'new' will not stand the test of time ;-)